ashr.go

```diff
@@ -0,0 +1,46 @@
+package nmea
+
+const (
+	// TypeASHR type for ASHR sentences
+	TypeASHR = "ASHR"
+)
+
+// ASHR represents RT300 proprietary roll and pitch sentence
+// https://gpsd.gitlab.io/gpsd/NMEA.html#_pashr_rt300_proprietary_roll_and_pitch_sentence
+//
+// Format: $-ASHR,hhmmss.sss,hhh.hh,T,rrr.rr,ppp.pp,xxx.xx,a.aaa,b.bbb,c.ccc,d,e*hh<CR><LF>
+// Example: $PASHR,085335.000,224.19,T,-01.26,+00.83,+00.10,0.101,0.113,0.267,1,0*07
+type ASHR struct {
+	BaseSentence
+	Time             Time
+	Heading          float64
+	TrueHeading      string
+	Roll             float64
+	Pitch            float64
+	Heave            float64
+	RollAccuracy     float64
+	PitchAccuracy    float64
+	HeadingAccuracy  float64
+	GPSUpdateQuality int64
+	INSStatus        int64
```

Comment:
GPSD calls this `IMU Status` and Trimble https://receiverhelp.trimble.com/oem-gnss/nmea0183-messages-pashr.html `IMU Alignment Status`

Reply:
> @Kaspermn33 could you link the RT300 manufacturer/product page?

Adding the RT 300 was a mistake, and I couldn't find anything under that name. 

I think it's an OXTS imu this will be used for.